lib/sms.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+module.exports = function (log, translator, templates, config) {
+  var nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+
+  var LINK = 'http://mzl.la/1HOd4ec'
+
+  return {
+    send: function (phoneNumber, senderId, messageId, acceptLanguage) {
+      log.info({
+        op: 'sms.send',
+        phoneNumber: phoneNumber,
```

Comment:
not sure about PII here, we either need to update the filter in Kibana or not log the phone number.
if we cannot log the full number, maybe just the area code?